### DISCUSSION TITLE
Release commit - is it possible

### DISCUSSION THREAD

**Discussion by grzesuav** [goreleaser/goreleaser#4677] [Q&A] [answered]
Hi, wasn't able to find an answer - is it possible to make a release commit - like via https://github.com/semantic-release/git - using `goreleaser` ? Idea is to update some files during release (set new version) and commit to repository

**Comment by caarlos0** [comment, marked as answer]
hey!

only tags... you can use semantic-release to generate the release commit, tag it, and then run goreleaser

**Reply by grzesuav** [reply]
Got it, I actually switched to go-sematic-release, which does not have this functionality yet.
